portfolio/src/main/java/com/google/sps/servlets/ServeCommentsServlet.java

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns comments stored in database. **/
+@WebServlet("/get-comments")
```

Comment:
I just realized this, sorry for the additional work.
You might want to follow a RESTful concept here.

RESTful here means that you would only have `/comments` (or similar) as a resource and use `GET` and `POST` to either get all comments or create a new one, instead of two different URLs `/get-comments` and `/add-comment`.

RESTful is a powerful tool and fits very well if you want to model database-like access (create, retrieve, update, or delete a resource aka [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete)) via HTTP(s).

I cannot finde a good explanation of RESTful API, but some pointers to RESTful best practices:
* https://florimond.dev/blog/articles/2018/08/restful-api-design-13-best-practices-to-make-your-users-happy/
* https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/